.github/workflows/weekly_pipeline.yml

```diff
@@ -0,0 +1,71 @@
+name: Weekly pipeline
+
+on:
+  push:
+    branches: [ main ]
+    paths: '.github/workflows/**'
+  pull_request:
+    branches: [ main ]
+    paths: '.github/workflows/**'
+  schedule:
+    - cron: '0 0 * * 0'  # Weekly on Sunday
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.head_ref || github.run_id }}
+  cancel-in-progress: true
+
+jobs:
+  build-test:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        ros_distro:
+          - rolling
```

Comment:
@nahueespinosa meta^2: stable distros do receive updates every once in a while. It might make sense to CI them as well, once a week JIC.

Reply:
Done in https://github.com/Ekumen-OS/beluga/pull/247/commits/08975fc2e3d1f096d18e2daf6e90f132fdb3ce77.